server/mongodb/actions/Analytics.ts

```diff
@@ -0,0 +1,141 @@
+import Analytics from "@server/mongodb/models/Analytics";
+import { Days } from "@/common_utils/types";
+
+export const getAnalyticsByID = async (userID: string) => {
```

Comment:
Create a type in `./common_utils/types.ts` for the Analytics record called `IAnalytics`. Use this type as the return type for this and all subsequent functions. Edit the MongoDB function call lines such that you assign a type to the return value, such as `const analytics = await Analytics.findOne<IAnalytics>({ userID });`